src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -37,6 +37,8 @@ public LinuxContainerEventGenerator(IEnvironment environment, Action<string> wri
 
         public static string DetailsEventRegex { get; } = $"{ScriptConstants.LinuxFunctionDetailsEventStreamName} (?<AppName>[^,]*),(?<FunctionName>[^,]*),\\\\\"(?<InputBindings>.*)\\\\\",\\\\\"(?<OutputBindings>.*)\\\\\",(?<ScriptType>[^,]*),(?<IsDisabled>[0|1])";
 
+        public static string AzureMonitorEventRegex { get; } = $"{ScriptConstants.LinuxAzureMonitorEventStreamName} (?<Level>[0-6]),(?<ResourceId>[^,]*),(?<OperationName>[^,]*),(?<Category>[^,]*),(?<RegionName>[^,]*),(?<Properties>[^,]*)";
```

Comment:
@brettsam  - does the property field need to be escaped? We do it for a few other fields today which come from customer code - like Details in FunctionsLogs

Reply:
What do you mean by escaped? We want to make sure customer logs can flow through here.

Reply:
I mean strings like these - "This string includes a quoted \"substring\" in the middle"

Reply:
I'm pretty sure .NET should be doing the escaping for you in that case. How do customers log details that contain quotes? Could you try that here?